builtins/amp-img.md

```diff
@@ -144,26 +144,43 @@ For the `<img>` tag in `HTML`, the `sizes` attribute is used in conjunction with
 
 See [Responsive images with srcset, sizes & heights](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/art_direction) for usage of `sizes` and `srcset`.
 
-**alt**
+### alt
 
 A string of alternate text, similar to the `alt` attribute on `img`.
 
-**attribution**
+### attribution
 
 A string that indicates the attribution of the image. For example, `attribution="CC courtesy of Cats on Flicker"`
 
-**height** and **width**
+### height and width
 
 An explicit size of the image, which is used by the AMP runtime to determine the aspect ratio without fetching the image.
 
-**common attributes**
+### common attributes
 
 This element includes [common attributes](https://amp.dev/documentation/guides-and-tutorials/learn/common_attributes) extended to AMP components.
 
-**data attributes**
+### data attributes
 
 Data attributes are copied from the `amp-img` element to the internal `img` element created by the component.
 
+[filter formats="email"]
+
+### Invalid AMP email attributes
+
+Security and privacy disallow the use of the following attributes on the AMP email format.
+
+- `[src]`
+- `[srcset]`
+- `lightbox`
+- `lightbox-thumbnail-id`
+- `object-fit`
+- `object-position`
+
+[tip type="read-on"]
+Enable a lightbox effect on `amp-img` in email by using the [`amp-image-lightbox`](../extensions/amp-image-lightbox/amp-image-lightbox.md) component with the `on` action.
+[/tip][/filter]
```

Comment:
Minor: Suggest adding a comment to closing tags so readers know what filter's being closed. E.g.
```
[/filter] <!-- formats="email" -->
```

Reply:
Good idea!